config/metadata/configMeta.yaml

```
@@ -1307,8 +1308,12 @@ groups:
         summary: controls whether to transmit traces in cache to remaining peers during cluster scaling event.
         description: >
           If `true`, Refinery will NOT forward live traces in its cache to the rest of the peers when peers join or leave the cluster.
-          By disabling this behavior, it can help to prevent disruptive bursts of network traffic when large traces with long `TraceTimeout`
-          are redistributed.
+          Disabling redistribution can help to prevent disruptive bursts of network traffic when large traces with long `TraceTimeout`
+          are redistributed. However, it will also cause data loss during scale up/down events.
+
+          Redistribution is intended to help prevent data loss and make whole trace decisions on scale up/down events.
+          This feature does incur additional resource usage. If the cluster does not have enough resource capacity headroom,
+          a redistribution event can cause the cluster to go into stress relief (if enabled).
```

Re-ordered the sentences and removed the precent disruptive bursts statement. All forwarded spans contribute to the additional resources being used during a scale up/down event, not just long traces.

```suggestion
Redistribution is intended to help prevent data loss and make whole trace decisions during cluster scale up/down events by redistributing stored spans to peers.
During scale down events, all stored spans are forwarded to the new owning Refinery peer instance so it can shut down without dropping any spans.
During scale up events, only stored spans that belong to another Refinery peer instance are forwarded so that instance can make whole trace decisions.
Refinery uses additional system resources during scale up/down events.
If the cluster does not have enough resource capacity headroom, a redistribution event can cause the cluster to go into stress relief (if enabled).
Disabling redistribution will incur data loss during scale up/down events.
```

---

Makes sense, I've reorganized per your suggestion!